packages/database/src/testUtilities/upsertDummyRecord.js

```diff
@@ -43,6 +43,7 @@ const processDefaultValue = defaultValue => {
 
 const generateDummyRecord = async (model, overrides = {}) => {
   const schema = await model.fetchSchema();
+  delete schema.m_row$;
```

Comment:
Would be worth adding a comment here, maybe mentioning to look in MaterializedViewLogDatabaseModel too. Could be confusing for people who don't know about analytics table process